phpunit/README.asc

```diff
@@ -68,24 +68,80 @@ mysql> FLUSH PRIVILEGES;
 4. fetch plugin Fusioninventory repository:
 +
 ----
-user@local:~$ git clone \
-    git@github.com:fusinv/fusioninventory-for-glpi.git \
-    ./glpi/plugins/fusioninventory
+user@local:~$ git clone https://github.com/fusioninventory/fusioninventory-for-glpi ~/glpi/plugins/fusioninventory
 ----
 
++
+5. prepare Glpi for the tests
++
+----
+user@local:~$ cd ~/fi_testsuite
+user@local:~$ cd glpi
+user@local:~$ composer install --no-dev
+user@local:~$ cd ..
+----
+
++
+5. prepare Glpi for the tests
++
+----
+user@local:~$ cd ~/fi_testsuite
+user@local:~$ cd glpi
+user@local:~$ composer install --no-dev
+user@local:~$ cd ..
+----
+
++
+6. install the FusionInventory plugin
++
+----
+user@local:~$ cd ~/fi_testsuite/glpi/plugins/fusioninventory/scripts/
+user@local:~$ php cli_install.php
+    ***** Update process of plugin FUSIONINVENTORY *****
+    Current FusionInventory version: 9.2+1.1
+    Version to update: 9.2+1.1
+    No migration needed.
+    Installing Plugin...
+    Migration Classname : CliMigration (0 seconds)
+    Update of plugin FusionInventory (0 seconds)
+    Change of the database layout - glpi_plugin_fusioninventory_agents (0 seconds)
+    Change of the database layout - glpi_plugin_fusioninventory_agentmodules (0 seconds)
+    Change of the database layout - glpi_plugin_fusioninventory_configs (0 seconds)
+    ...
+    ...
+    Add Crontasks (4 seconds)
+    Install Done
+    Activating Plugin...
+    Activation Done
+    Loading Plugin...
+    Load Done...
+
 
 == Usage
 
 This set of testsuites are meant to be executed with +ant+.
 
+To install Apache ant :
+
+1. sudo apt-get install ant
```

Comment:
This will not work on many systems... Better to add  link to the ant website and say something more generic

Reply:
Ok